dev/NavigationView/NavigationView.cpp

```diff
@@ -1779,10 +1779,90 @@ void NavigationView::OnNavigationViewItemKeyDown(const winrt::IInspectable& send
     }
 }
 
+void NavigationView::ArrowKeyNavigationPolyfill(const winrt::NavigationViewItem& nvi, const winrt::KeyRoutedEventArgs& args)
+{
+    switch (args.Key())
+    {
+    case winrt::VirtualKey::Down:
+        if (auto const ir = GetParentItemsRepeaterForContainer(nvi))
+        {
+            if (ir != m_topNavRepeater.get())
```

Comment:
>ir != m_topNavRepeater.get() [](start = 16, length = 28)

would !IsTopNavigationView() be more readable ?

Reply:
We still want to achieve this behavior if the overflow menu in topnavigationview has enough items to scroll. Its just the horizontal itemsrepeater we dont care about. I'll add a comment to make that more clear.

---
In reply to: [367156075](https://github.com/microsoft/microsoft-ui-xaml/pull/1850#discussion_r367156075) [](ancestors = 367156075)